test/polytopes.jl

```diff
@@ -870,4 +870,49 @@
       └─ Point(0.0, 0.0, 1.0)"""
     end
   end
+
+  @testset "Simplex" begin
+    @testset "construction" begin
+        pts = (Point(1., 2., 3.), Point(1., 2., 4.));
+        pts32 = convert.(Point{3,Float32}, pts)
```

Comment:
Please use the T, P2, P3 constants defined in the tests. The tests are run with single and double precision if you use these constants.